### DISCUSSION TITLE
Multiple proxy host responding on ports 80/443 and others (not HTTP standard)

### DISCUSSION THREAD

**Discussion by atscoders** [NginxProxyManager/nginx-proxy-manager#2362] [Q&A]
Hi everyone,
my docker-compose has this ports section:

ports:
      # Public HTTP Port
      - '80:80'
      # Admin Web Port
      - '81:81'
      # Public HTTPS Port
      - '443:443'
      # Public not standard HTTP Ports
      - '8080:8080'
      # Public not standard HTTPS Ports
      - '8443:8443' 

NPM should have multiple proxy hosts, some responding to standard HTTP/HTTPS ports and others responding ONLY on not standard HTTP/HTTPS ports (in this case port 8080 and 8443).
How can I handle this requirement? Should I use "Custom Nginx Configuration" in Advanced tab or should I create another config file?
I just tried to add "listen 8443 ssl http2;" on "Custome Nginx Configuration", but the service responds correctly on port 8443, but also on ports 443.
I hope I have explained my need well and thank you in advance.

**Comment by KajKunstheim** [comment]
You have to write your own custom proxy configuration. The configurations gerated by the webinterface always listen on port 80/443. Would be great if the devs would add a feature to specify listening ports in the webinterface. You have my vote for that :-)

Cheers!
